lib/sisimai/bite/email/verizon.rb

```diff
@@ -0,0 +1,265 @@
+module Sisimai::Bite::Email
+  module Verizon
+    # Sisimai::Bite::Email::Verizon parses a bounce email which created by
+    # Verizon Wireless.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Verizon.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'received'  => %r/by .+[.]vtext[.]com /,
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.
* Layout/SpaceAroundOperators: Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->